docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -0,0 +1,941 @@
+:branch:          1.5
+:ls_version:		  1.5.0.beta1
+
+= How to write your own Logstash {plugintype} plugin
+
+This HOWTO will walk you through the process of developing a new {plugintype}
+for Logstash. The workflow has changed as of version 1.5 of Logstash, so you'll
+want to follow these new and improved steps to contribute a new plugin!
+
+
+{introfooter}
+
+Logstash is written in Ruby.  If you are unfamiliar with the Ruby programming
+language, an excellent quick-start guide is at
+https://www.ruby-lang.org/en/documentation/quickstart/.
+
+== Get started
```

Comment:
should we move 
>  For this example, we are going to examine the example input plugin at https://github.com/logstash-plugins/logstash-input-example/

to first line of:

## Get started

-------------------------------------------------------------

Seems more coherent? this is currently in introfooter